extension/llm/export/export_llm.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+"""
+Export an LLM with ExecuTorch. Currently follows the following steps:
+1. Instantiate our custom PyTorch transformer definition from examples/llama/models/llama_transformer.py.
```

Comment:
Don't mention implementation details in the docblock. 

If it's a public API, the docblock should contain description of the contract.

Reply:
I move this  this information to a README